src/components/binding/bindApplicationForm.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+angular.module('openshiftCommonUI').component('bindApplicationForm', {
+  controllerAs: 'ctrl',
+  bindings: {
+    allowNoBinding: '<?',
+    createBinding: '=',
+    applicationName: '=',
+    formName: '=',
+    serviceClasses: '<',
+    orderedServiceInstances: '<',
```

Comment:
guess i didnt hit save on my comment....
orderedServiceInstances is redundant since a service instance doesnt exist unless a service class was provisioned (ordered). Lets just call this serviceInstances